lib/streamlit/elements/multiselect.py

```diff
@@ -104,6 +104,26 @@ def _check_and_convert_to_indices(
     return [opt.index(value) for value in default_values]
 
 
+def get_default_count(default: Union[Sequence[Any], Any, None]) -> int:
+    if default is None:
+        return 0
+    if not is_iterable(default):
+        return 1
+    return len(cast(Sequence[Any], default))
+
+
+def get_options_message(current_selections: int, max_selections: int):
```

Comment:
This name isn't very descriptive: would something like `over_max_selections_msg` be better?